src/main/java/com/gradle/Overrides.java

```diff
@@ -52,6 +56,7 @@ void configureGradleEnterpriseOnGradle4(BuildScanExtension buildScan) {
         booleanSysPropertyOrEnvVariable(GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER, providers).ifPresent(buildScan::setAllowUntrustedServer);
     }
 
+    @SuppressWarnings("DataFlowIssue")
```

Comment:
Why is this suppressed? What issues with "data flow" might there be?

Reply:
A null check for the remote URL.